.github/workflows/pr-manager.yml

```
@@ -0,0 +1,21 @@
+name: "Pull Request Manager"
+on:
+  schedule:
+    - cron: '0 * * * *'
+
+jobs:
+  stale:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Process stale issues and PRs
```

```suggestion
      - name: Process stale PRs
```
Added this suggestion because we don't have an issue configuration in this PR.

But I wonder if we should add configuration for the issues as well.

---

@djabarovgeorge I think lets keep it to PRs for the moment then if we like it then we add it to issues.